main.go

```diff
@@ -394,7 +421,7 @@ func main() {
 		ControllerName:      controllerDeploymentName,
 		HasNamespaceAccess:  clusterScopeMode,
 		RegistryMap:         registryMap,
-	}).SetupWithManager(mgr, enableIsvcWatch, runtimeControllerEvents); err != nil {
+	}).SetupWithManager(mgr, enableIsvcWatch, runtimeControllerEvents, enableCSRWatch); err != nil {
```

Comment:
I think we probably want to add this as a ServingRuntimeReconciler field instead and also use it to control whether we Get/List the CSRs in various places (instead of using `HasNamespaceAccess` for that).